src/components/SideBar.vue

```diff
@@ -144,6 +191,10 @@ export default {
   },
   mounted: function () {
     EventBus.on('PopoverActionClick', (payLoad) => {
+      /**
+       * This event is emitted when user clicks the gallery image.
```

Comment:
The ActionClick event is emitted when the image is clicked on or the button below the image is clicked on.

Reply:
Thanks for the review. This is updated now.